packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -161,7 +161,7 @@ export async function createNewEmptyFluidFile(
                 }
                 event.end({
                     headers: Object.keys(headers).length !== 0 ? true : undefined,
-                    ...fetchResponse.commonSpoHeaders,
+                    ...fetchResponse.telemetryProps,
```

Comment:
I'd personally prefer an old name. "Telemetry props" is the type, it feels like Hungarian notation. "Spo headers" is what they represent.
If you want to rename, then likely you want to articulate the purpose, i.e., "fetchTelemetryPropsToLog" or something like that.

Reply:
People are putting other stuff not headers on there